src/party/sda.md

```diff
@@ -4,76 +4,81 @@ index: true
 
 # Simple data analysis
 
-[Simple data analysis](https://nshiab.github.io/simple-data-analysis/), a library maintained by computational journalist Nael Shiab, purports to be “a high-performance and convenient solution in JavaScript for data analysis. It's based on DuckDB and inspired by Pandas (Python) and the Tidyverse (R).”
+[Simple data analysis](https://github.com/nshiab/simple-data-analysis), a
+library maintained by computational journalist Nael Shiab, purports to be “a
+high-performance and convenient solution in JavaScript for data analysis. It's
+based on DuckDB and inspired by Pandas (Python) and the Tidyverse (R).”
 
 ```js echo
-import {SimpleDB} from "npm:simple-data-analysis@2";
+import { SimpleWebDB } from "https://esm.sh/jsr/@nshiab/simple-data-analysis@3.15.3/web";
```

Comment:
I'd want this to work:

```
import {SimpleWebDB} from "jsr:@nshiab/simple-data-analysis@3/web";
```

unfortunately it fails with `Missing "./web" specifier in "@jsr/nshiab__simple-data-analysis" package`, which may be an issue with the package or with framework (?). I'm not sure how to fix it yet.


Note that ideally, I'd want this:
```
import {SimpleWebDB} from "jsr:@nshiab/simple-data-analysis@3";
```

Reply:
I would also want to use a self-hosted version of DuckDB, instead of loading it from jsdeliver

Reply:
From what I see in the console when I switch to `import { SimpleWebDB } from "jsr:@nshiab/simple-data-analysis@3.15.3/web";`, it's a problem with the library. I'll investigate and come back to you!